src/Futurice/Time/Month.hs

```diff
@@ -112,6 +113,9 @@ instance Show Month where
 -- TODO write Read instance to match above Show instance
 
 instance Hashable Month
+instance Csv.ToField Month
```

Comment:
How does the generic version look? Is the generic version output same as Csv.toField . show?

Reply:
It appears to be exactly identical to the version I did manually.